packetbeat/_meta/beat.full.yml

```diff
@@ -432,23 +438,22 @@ packetbeat.protocols.nfs:
 #
 # Process matching is optional and can be enabled by uncommenting the following
 # lines.
-#
-#packetbeat.procs:
-#  enabled: false
-#  monitored:
-#    - process: mysqld
-#      cmdline_grep: mysqld
-#
-#    - process: pgsql
-#      cmdline_grep: postgres
-#
-#    - process: nginx
-#      cmdline_grep: nginx
-#
-#    - process: app
-#      cmdline_grep: gunicorn
+packetbeat.procs:
+  enabled: false
+  monitored:
+    - process: mysqld
+      cmdline_grep: mysqld
+
+    - process: pgsql
+      cmdline_grep: postgres
+
+    - process: nginx
+      cmdline_grep: nginx
+
+    - process: app
+      cmdline_grep: gunicorn
 
-# Uncomment the following if you want to ignore transactions created
+# Set the following to true if you want to ignore transactions created
```

Comment:
@tsg The title kind of indicates that this belongs to Monitored processes, but I think it doesn't? Perhaps we should just have a title for "General" packetbeat config and then it would also belong on the top with interfaces or interfaces at the bottom. WDYT?